packaging/rpm/SPECS/nvidia-container-toolkit.spec

```diff
@@ -64,6 +79,9 @@ fi
 %files
 %license LICENSE
 %{_bindir}/nvidia-container-runtime-hook
+%config /etc/systemd/system/nvidia-cdi-refresh.service
+%config /etc/systemd/system/nvidia-cdi-refresh.path
```

Comment:
Does defining these as configs mean that a remove doesn't remove them?

Reply:
umm yes, I was following what I saw on the mig-parted install script, but I guess here we want a different behaviour